### PR TITLE
Amendments and Supporting information

### PR DESCRIPTION
**Pull request by einsteinagogo**
Change: Some typos, uppercase
Added: New Talk IT Author and British Beekeeper.
Added: Additional supporting information Type 1 and Type 2 hypervisors, and more examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 13. [Cyber Security Resources](#cyber)
 14. [Other Interesting Stuff](#misc)
 15. [Andrew Hancock, Department of Computer Science & Technology - Systems](#andrewhancock)
-16. [Hypervisors and non VMWare VM Stuff](#vmstuff)
+16. [Hypervisors and non VMware VM Stuff](#vmstuff)
```
Comment (closebracket) [inline README.md]: This category isn't correct if we're adding VMware to the category. Please rename and resubmit.

File: README.md [hunk 2]
```diff
@@ -325,10 +327,33 @@ Andrew is the VMware Specialist in Computer Science & Technology, responsible fo
 130. [HOW TO: Install and Connect a VMware vCenter Server 7.0 in linked mode.](https://www.experts-exchange.com/articles/35052/HOW-TO-Install-and-Connect-a-VMware-vCenter-Server-7-0-in-linked-mode.html)
 131. [HOW TO: BOOT VMware vSphere Hypervisor 7.0 (ESXi 7.0 ARM) from an iSCSI LUN for the Raspberry Pi 4](http://vexpert.me/iSCSI-BOOT-from-ESXi-ARM)
 
-## Hypervisors and vms <a name="vmstuff"></a>
+## Hypervisors and virtual machines  <a name="vmstuff"></a>
+
+Ontop of the great VMware resources above there are other hypervisors for running containers and virtual machines.
+
+Hypervisors are classified as Type 1 or Type 2 hypervisors. Type 1 are often known as bare metal hypervisors. It's installed on bare metal, e.g. it is not installed as an application on top of an operating system. A Type 2 hypervisor is an application which is installed on the operating system.
```
Comment (closebracket) [inline README.md:334]: Great!

File: README.md [hunk 3]
```diff
@@ -325,10 +327,33 @@ Andrew is the VMware Specialist in Computer Science & Technology, responsible fo
...
+Type 2 Hypervisors can be SLOW.  In most reviews and experience, they perform at roughly 30-40% hardware capability.  That means an Guest Operating Systems in a virtual machine hosted on a Type 2 hyperviosor will likely perform at best like it has an 800 MHz CPU if you have 2 GHz physical CPU.
```
Comment (closebracket) [inline README.md:335]: Great!

File: README.md [hunk 4]
```diff
@@ -325,10 +327,33 @@ Andrew is the VMware Specialist in Computer Science & Technology, responsible fo
...
 
-## Hypervisors and vms <a name="vmstuff"></a>
+## Hypervisors and virtual machines  <a name="vmstuff"></a>
+
+Ontop of the great VMware resources above there are other hypervisors for running containers and virtual machines.
+
+Hypervisors are classified as Type 1 or Type 2 hypervisors. Type 1 are often known as bare metal hypervisors. It's installed on bare metal, e.g. it is not installed as an application on top of an operating system. A Type 2 hypervisor is an application which is installed on the operating system.
+Type 2 Hypervisors can be SLOW.  In most reviews and experience, they perform at roughly 30-40% hardware capability.  That means an Guest Operating Systems in a virtual machine hosted on a Type 2 hyperviosor will likely perform at best like it has an 800 MHz CPU if you have 2 GHz physical CPU. 
+
+If you use a Type 1 Hypervisor, you get much better performance (based on experience and reviews) typically get 80-90% hardware capability - so that same virtual machines hosted on the same 2 GHz CPU should operate more like it has a 1.6 GHz CPU instead of 800 Mhz.
```
Comment (closebracket) [inline README.md:337]: Great!

File: README.md [hunk 5]
```diff
@@ -325,10 +327,33 @@ Andrew is the VMware Specialist in Computer Science & Technology, responsible fo
...
+6. [Microsoft Windows Server 2019](https://www.microsoft.com/en-gb/windows-server) - Hyper-V can be enabled as a role in Windows Server. This is a Type 1 hypervisor. - Available for free if a Student of The Department of Computer Science and Technology.
+7. [Microsoft Hyper-V Server](https://www.microsoft.com/en-us/evalcenter/evaluate-hyper-v-server-2019_) - Not to be confused with Windows Server. This is a Type 1 Hypervisor, and is just server core Hyper-V with no GUI. - Available for free if a Student of The Department of Computer Science and Technology.
+8. [Microsoft Windows 10 Client Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/about/) - Hyper-V can be enabled in Windows 10 Pro and above. This is a Type 1 Hypervisor. - Available for free if a Student of The Department of Computer Science and Technology.
+9. [VMware vSphere HYpervisor (ESXi)](https://my.vmware.com/en/web/vmware/evalcenter?p=free-esxi7) - Available for free, or full version - Available for free if a Student of The Department of Computer Science and Technology.
```
Comment (closebracket) [inline README.md]: Brilliant. Thanks for adding.

File: README.md [hunk 6]
```diff
@@ -325,10 +327,33 @@ Andrew is the VMware Specialist in Computer Science & Technology, responsible fo
...
 
-1. [Proxmox](http://www.proxmox.com) - A Free version of the comerial proxmox hypervisor
-2. [Nutanix CE](https://www.nutanix.com/products/community-edition) - A free version of the nutanix HCI platform
+Type 2
 
+1. [Oracle Virtualbox](https://www.virtualbox.org/) - A Free alternative to VMware Workstation for all platforms
+2. [VMware Fusion for Apple Mac](https://www.vmware.com/uk/products/fusion.html) - Type 2 Hypervisor for Apple Mac - Available for free if a Student of The Department of Computer Science and Technology.
+3. [VMware Workstation](https://www.vmware.com/uk/products/workstation-pro.html) - Type 2 Hypervisor for Linux and Windows - Available for free if a Student of The Department of Computer Science and Technology.
+4. [VMware Player](https://www.vmware.com/uk/products/workstation-player.html) - Type 2 Hypervosor for Linux and Windows - Available for free if a Student of The Department of Computer Science and Technology.
+5. [Parallels Desktop 16 for Mac](https://www.parallels.com/uk/products/desktop/) - Type 2 Hypervisor for Mac.
```
Comment (closebracket) [inline README.md:359]: Brilliant! Thanks for adding!

**Comment by closebracket** [comment]
Please complete requested changes and this will be merged.

**Comment by einsteinagogo** [comment]
> Please complete requested changes and this will be merged.

what changes are required ?

**Comment by sbrl** [comment]
Oops @closebracket, if you tried to leave a review it didn't go through :slightly_smiling_face:

**Comment by closebracket** [comment]
My bad. Using the mobile app. Thought adding the comments was enough!

**Comment by closebracket** [comment]
> > Please complete requested changes and this will be merged.
> 
> what changes are required ?

Pushed now. Sorry about that.

**Comment by einsteinagogo** [comment]
Yes, I've just spotted this. 

Category corrected : - Hypervisors and Virtual Machines, the main category was already changed, just the main link.

File: README.md (hunk 1 above)
Comment (einsteinagogo) [reply README.md]: Completed.

**Comment by admgecko** [comment]
Looks good to me

**Pull request merged**